docs/manual/docs/cmd/spo/app/app-uninstall.md

```diff
@@ -15,6 +15,7 @@ Option|Description
 `--help`|output usage information
 `-i, --id <id>`|ID of the app to retrieve information for
 `-s, --siteUrl <siteUrl>`|Absolute URL of the site to uninstall the app from
+`--scope [scope]`|Scope of the app catalog: tenant|sitecollection. Default tenant
```

Comment:
`tenant|sitecollection` should be wrapped in backticks. Currently, the pipe creates a new table column and some of the information is cut off.